src/vs/editor/contrib/suggest/suggestController.ts

```diff
@@ -406,7 +406,13 @@ registerEditorCommand(new SuggestCommand({
 registerEditorCommand(new SuggestCommand({
 	id: 'selectLastSuggestion',
 	precondition: ContextKeyExpr.and(SuggestContext.Visible, SuggestContext.MultipleSuggestions),
-	handler: c => c.selectLastSuggestion()
+	handler: c => c.selectLastSuggestion(),
+	kbOpts: {
+		weight: weight,
+		kbExpr: EditorContextKeys.textFocus,
+		primary: KeyCode.End,
+		secondary: [KeyCode.End],
```

Comment:
Use `secondary` only when the key code is different from `primary` 🤓

Reply:
Fixed